cypress/integration/free/admin.spec.ts

```diff
@@ -195,20 +460,37 @@ describe(
     describe("Manage software page", () => {
       beforeEach(() => {
         cy.loginWithCySession("anna@organization.com", "user123#");
+        cy.intercept("GET", "/api/latest/fleet/config", getConfig).as(
+          "getIntegrations"
+        );
         cy.visit("/software/manage");
+        cy.wait("@getIntegrations").then((configStub) => {
+          console.log(JSON.stringify(configStub));
```

Comment:
Do we need to keep this log?

Reply:
I left them as a reminder that we "received" mock data responses from the API